CHANGELOG.md

```diff
@@ -6,11 +6,16 @@ IMPROVEMENTS:
 * CRDs: update the CRD versions from v1beta1 to v1. [[GH-883](https://github.com/hashicorp/consul-helm/pull/883)]
 * Enterprise: support applying Consul Enterprise license when security context defaults to non-root users. [[GH-880](https://github.com/hashicorp/consul-helm/pull/880)]
 * Sync Catalog: add new `syncCatalog.extraLabels` Helm value for configuring labels on sync catalog pods. [[GH-892](https://github.com/hashicorp/consul-helm/pull/892)]
+* Connect: Support high availability of the connect-inject deployment. [[GH-903](https://github.com/hashicorp/consul-helm/pull/903)]
 
 BREAKING CHANGES:
 * Minimum Kubernetes versions supported is 1.16+. [[GH-883](https://github.com/hashicorp/consul-helm/pull/883)]
 * Connect: `-enable-health-checks-controller`, `-health-checks-reconcile-period`, `-cleanup-controller-reconcile-period` have been removed
-  and are no longer supported as the controllers have been replaced by the endpoints controller. [[GH-892](https://github.com/hashicorp/consul-helm/pull/899)]
+  and are no longer supported as the controllers have been replaced by the endpoints controller.
+  The associated helm values have also been removed: `global.connectInject.healthChecks` and `global.connectInject.cleanupController`. [[GH-899](https://github.com/hashicorp/consul-helm/pull/899)]
```

Comment:
I think this change should be in the context of the helm values rather than command flags since it's for the helm chart. Maybe just rephrase the sentence below saying which helm values have been removed.